packit/cli/dist_git_init.py

```diff
@@ -216,6 +221,31 @@ def init(
         logger.warning("At least one job needs to be defined!")
         return
 
+    if upstream_git_url and upstream_git_url_command:
+        click.echo(
+            "Only one of --upstream-git-url or --upstream-git-url-command can be specified.",
+            err=True,
+        )
+        return
+
+    if not (upstream_git_url or upstream_git_url_command):
+        click.echo(
+            "One of --upstream-git-url or --upstream-git-url-command needs to be specified.",
+            err=True,
+        )
+        return
+
+    click.echo(
+        f"Running the following command to find upstream git URL: {upstream_git_url_command}",
+    )
+    result = commands.run_command(
+        upstream_git_url_command,
+        output=True,
+        cwd=path_or_url.working_dir,
+    )
+    upstream_git_url = result.stdout.strip()
+    click.echo(f"Found the following URL: {upstream_git_url}")
```

Comment:
shouldn't this be executed only when `upstream_git_url` is None (=> or `upstream_git_url_command` is not None)?

Reply:
Oh, oh...🤦

Reply:
Give me a sec to send a fix.